src/client/worker.rs

```diff
@@ -45,3 +46,52 @@ pub fn print_worker_info(workers: Vec<WorkerInfo>, gsettings: &GlobalSettings) {
         ]);
     assert!(print_stdout(table).is_ok());
 }
+
+pub fn print_worker_details(worker: WorkerDetail, gsettings: &GlobalSettings) {
```

Comment:
Please do not duplicate print_worker_configuration. If you need to modify it, try to refactor by introducing a new function that is shared by both functions. However as it seems it may be just a one function now